README.md

```diff
@@ -31,20 +31,33 @@ The data for contamination spread scenarios is pre-computed and coupled with inf
 1. Visualization of Weisi/Alessio's Neural Network's decision making process for reducing a water network to a set of critical nodes
 2. A version of the app that water networks can input their own water networks and contamination scenario data to, by uploading files in a format specified by the app
 
-Bokeh App
+Run the App Locally
 -------
 
 *Tested with Python 3.6 and 3.7*
 
 1. Clone the repository: `git clone https://github.com/alan-turing-institute/chance-water-distribution`
 2. Install the python requirements: `pip install -r requirements.txt`
 3. Download data modules: `git submodule update --init --recursive`
-4. Run bokeh server from top dir of the repo: `bokeh serve --show water`
+4. Run flask server from top dir of the repo: `python water/main.py`
+
+Deploying the App
+-------
+
+When deploying the app Flask's built in server should not be used. Suggestions
+for WSGI servers that can be used with Flask can be found
+[here](https://flask.palletsprojects.com/en/1.0.x/deploying/). For example,
+gunicorn can be used by
+```
+pip install gunicorn
+cd water
+gunicorn -w 4 main:app
+```
 
 ### Docker version
 
 1. Pull from Docker Hub: `docker pull turinginst/chance-water`
-2. Run the container `docker run -p 5006:5006 turinginst/chance-water`
-3. Open http://localhost:5006/water in a browser
+2. Run the container `docker run -p 8000:8000 turinginst/chance-water`
+3. Open http://localhost:8000 in a browser
```

Comment:
I think might be important to specify this step for the local version as well. The terminal suggests to go to http://127.0.0.1:8000/ which doesn't work properly, but http://localhost:8000 does

Reply:
On your system? 127.0.0.1 and localhost *should* be the same.

I agree it should also be mentioned with the local instructions too.

Reply:
Yes on my laptop they were different for some reason, not sure why that would be